roles/create_mce_crs/files/07_create-cluster-deployment-template.yml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: v1
+kind: Template

```

Comment:
Is there a need to use the k8s templates instead of using Jinja? This adds another form of template into the codebase which seems unnecessary. 

You are even passing the values to the templates from ansible so why not use jinja?

Which also seems to lead a lot of tasks adding these templates to the cluster.

Reply:
I wanted use templates because some of these custom resources that will be deployed requires parameters so I found this way but if Jinja is easier way, we can take a look at it. But this template method was not that hard and not lead to lot of tasks, it was easy :)